docs/HowTo/mirror-eurolinux-locally.md

```diff
@@ -0,0 +1,203 @@
+    license. You can read about that below.
+
+The official way to mirror EuroLinux repositories is the following:
+
+- You need a proper subscription, like EuroMan or Gold Key
+- EuroLinux engineer will provide you with SSL certificates that we will name
+  `repo.key` and `repo.crt` and CA that we will name `ca.crt`
+
+
+With repokeys residing in `/etc/yum-mirror-config/` directory create the
+following yum.config - example below is for EuroLinux 7 x86_64
+
+
+```ini
+[main]
+cachedir=/var/cache/yum/mirror/$basearch/$releasever
+keepcache=0
+debuglevel=2
+logfile=/var/log/mirror-yum.log
+exactarch=0
+obsoletes=0
+gpgcheck=0
+plugins=0
+reposdir=/dev/null
+
+[eurolinux-os-7]
+name=el7_x86_64_os
+baseurl=https://cdn.euro-linux.com/dist/eurolinux/server/7/x86_64/os/
+sslclientkey=/etc/yum-mirror-config/repo.key
+sslclientcrt=/etc/yum-mirror-config/repo.crt
+sslcacert=/etc/yum-mirror-config/ca.crt
+
+[eurolinux-updates-7]
+name=el7_x86_64_updates
+baseurl=https://cdn.euro-linux.com/dist/eurolinux/server/7/x86_64/updates/
+sslclientkey=/etc/yum-mirror-config/repo.key
+sslclientcrt=/etc/yum-mirror-config/repo.crt
+sslcacert=/etc/yum-mirror-config/ca.crt
+```
+
+After saving this file to `/etc/yum-mirror-config/mirror_yum.conf` you might download the
+repositories to the `/repos` directory with the following command.
+```
+reposync -d -m --download-metadata -c /etc/yum-mirror-config/mirror_yum.conf -p /repos
+```
+
+When the download finish, the next step is to create repodata and enable
+groups.
+
+```bash
+cd /repos/eurolinux-os-7/; createrepo . -g comps.xml
+cd /repos/eurolinux-updates-7/; createrepo . -g comps.xml
+```
+
+!!! info Erratas
+    Enabling updateinfo (erratas information) is a little bit tricky, because
+    firstly you have to find the newest updateinfo, unpack it, then invoke
+    modifyrepo script. It can be automated with the script below.
+
+```bash
+REPO_DIR=/repos/eurolinux-os-7/
+unset -v LAST_UI
+# finding the newest file
+for file in "$REPO_DIR"/*updateinfo.xml.gz; do
+  [[ "$file" -nt "$LAST_UI" ]] && LAST_UI=$file
+done
+# unpacking to updateinfo.xml file
+sudo gunzip -c "$LAST_UI"  > "$REPO_DIR/updateinfo.xml"
+# Depending on the system - some has modifrepo.py script some has "normal" command
+/usr/share/createrepo/modifyrepo.py "$REPO_DIR/updateinfo.xml"  "$REPO_DIR/repodata" || modifyrepo "$REPO_DIR/updateinfo.xml"  "$REPO_DIR/repodata"
+
+REPO_DIR=/repos/eurolinux-updates-7/
+unset -v LAST_UI
+for file in "$REPO_DIR"/*updateinfo.xml.gz; do
+  [[ $file -nt $LAST_UI ]] && LAST_UI=$file
+done
+sudo gunzip -c "$LAST_UI"  > "$REPO_DIR/updateinfo.xml"
+# Depending on the system - some has modifrepo.py script some has "normal" command
+/usr/share/createrepo/modifyrepo.py "$REPO_DIR/updateinfo.xml"  "$REPO_DIR/repodata" || modifyrepo "$REPO_DIR/updateinfo.xml"  "$REPO_DIR/repodata"
+```
+
+This short manual concludes mirroring EuroLinux locally.
+
+## Mirroring EuroLinux 7 **not supported way**
```

Comment:
Ja bym dał _- **unsupported way**_: https://ell.stackexchange.com/questions/91033/unsupported-vs-not-supported, ale może być.